### PR TITLE
Updating of CDM enums

### PR DESCRIPTION
**Pull request by manucarreramoreno**
This PR contains the annotation of some CDM enums to their correspondent FpML coding scheme, as well as the update on CapacityUnitsEnum, with the new values on FpML

### CODE REVIEW

**Review by tomhealey-icma — DISMISSED** [review]

**Comment by manucarreramoreno** [comment]
[Release-Notes-CDMEnums.md](https://github.com/finos/common-domain-model/files/10943761/Release-Notes-CDMEnums.md)

File: rosetta-source/src/main/rosetta/event-common-enum.rosetta [hunk 1]
```diff
@@ -126,6 +128,7 @@ enum CorporateActionTypeEnum: <"The enumerated values to specify the origin of a
 	Liquidation <"Corporate action triggered by a liquidation. When a business or firm is terminated or bankrupt, its assets are sold (liquidated) and the proceeds pay creditors. Any leftovers are distributed to shareholders. The value maps closely to the ISO code (LIQU) defined as a distribution of cash, assets or both. Debt may be paid in order of priority based on preferred claims to assets specified by the security.">
 
 enum CreditEventTypeEnum: <"Represents the enumerated values to specify a credit event type.">
+    [docReference ISDA FpML_Coding_Scheme schemeLocation "http://www.fpml.org/coding-scheme/credit-event-type"]
```
Comment (hugohills-regnosys) [inline rosetta-source/src/main/rosetta/event-common-enum.rosetta:131]: The CDM enum does not match the scheme, please update

File: rosetta-source/src/main/rosetta/observable-asset-enum.rosetta [hunk 2]
```diff
@@ -5,7 +5,7 @@ import cdm.base.*
 import cdm.mapping.config.*
 
 enum InformationProviderEnum: <"The enumerated values to specify the list of information providers.">
-	//[docReference ISDA FpML_Coding_Scheme schemeLocation "http://www.fpml.org/coding-scheme/information-provider"]
+    [docReference ISDA FpML_Coding_Scheme schemeLocation "http://www.fpml.org/coding-scheme/information-provider"]
```
Comment (hugohills-regnosys) [inline rosetta-source/src/main/rosetta/observable-asset-enum.rosetta:8]: The CDM enum does not match the scheme, please update

**Review by hugohills-regnosys — APPROVED** [review]

**Pull request merged**